WordPress/Docs/Security/SafeRedirectStandard.xml

```diff
@@ -0,0 +1,19 @@
+<documentation title="Safe Redirect">
+    <standard>
+    <![CDATA[
+    Safe redirects must be used.
```

Comment:
I'd like to see this expanded - _why_ must safe redirects be used? What is a _safe_ redirect? Why does a non-safe redirect even exist? Where can I find out more about this concept? etc.